README.md

```diff
@@ -1,114 +1,80 @@
 [![Open in GitHub Codespaces](
   https://img.shields.io/badge/Open%20in%20GitHub%20Codespaces-333?logo=github)](
   https://codespaces.new/dwave-examples/employee-scheduling?quickstart=1)
-[![Linux/Mac/Windows build status](
+<!-- [![Linux/Mac/Windows build status](
   https://circleci.com/gh/dwave-examples/employee-scheduling.svg?style=shield)](
-  https://circleci.com/gh/dwave-examples/employee-scheduling)
+  https://circleci.com/gh/dwave-examples/employee-scheduling) -->
 
-# Building an Employee Schedule
+# Employee Scheduling Demo
 
-Building a schedule for employees can be an extremely complex optimization
-problem in which managers must balance employee preferences against schedule
-requirements. In this example, we show how a discrete quadratic model (DQM) 
-and a constrained quadratic model (CQM) can be used to model this problem and
-how the hybrid solvers available in Leap can optimize over these competing 
-scheduling and preference needs.
+Employee scheduling is a common industry problem that can often become complex
+due to real-world constraints and requirements. In this example, we demonstrate
+a scheduling scenario with a variety of employees and rules.
```

Comment:
```suggestion
a scheduling scenario with a variety of employees and rules.

![Screen Image](assets/demo_image.png)
```

In keeping with our recently updated [example template](https://github.com/dwave-examples/template), this is the recommended position for your screenshot.